repo/packages/K/kubernetes/2/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "kubernetes",
+  "version": "v1.0.6-v0.6.5-alpha",
+  "scm": "https://github.com/mesosphere/kubernetes-mesos",
+  "maintainer": "support@mesosphere.io",
+  "description": "Manage a cluster of Linux containers as a single system to accelerate Dev and simplify Ops.",
```

Comment:
This description has always bothered me. I think it's the tense. It sounds like it's telling the user to do something, rather than describing the package. I seem to recall there's some place the cli displays the description w/o the package name too, like a "long name" vs "short name". But I can't remember where.